### ISSUE TITLE
Discuss and improve tutorial

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2521]
There was a recent [Twitter thread](https://twitter.com/alexhaagaard/status/1375394748605808640?s=20) where the Project Sidewalk tutorial was criticized for improperly training users about sidewalk accessibility. Though written bitingly, the thread has some great points that we should discuss.

![image](https://user-images.githubusercontent.com/1621749/112635500-26aa4c00-8df9-11eb-9ee1-807301959707.png).

For example, they argue strongly that a perpendicular facing ramp, which places a person with a disability into the flow of traffic is "one of the scariest access problems" they face on a regular basis.

@misaugstad, we should chat about how to improve the tutorial experience to address current limitations (not just captured by this person's perspective but more broadly, of course).

**Comment by misaugstad** [comment]
Agreed. I saw that later in the thread, they suggested that they would have rated it as a 4 or 5 severity depending on how busy the intersection was. So a severity of 3 may be overly generous.

I agree that we should consider this feedback (and others; I can look through the notes submitted through the "Feedback" button as well) and update the tutorial accordingly.
